CrisisTextLine/La-Sape
# mike/funky-apple-change/AT-798

## Basic Info
#### What's this PR do?
changes all 'Funky Apples' references to 'Escalations'
Also introduces different options for Funky Apples/Escalations

also this PR: https://github.com/CrisisTextLine/app/pull/1779
#### What are the relevant tickets?
https://admin.crisistextline.org/jira/browse/AT-798
#### Where should the reviewer start?

## In Depth
#### Any refactoring done (or necessary)?

#### Any background context you want to provide?

## Performance
#### Are there enforced limits on all of the queries? Anything that will output over 10kb?

## Deployment
#### Integration testing steps for pre-deploy checklist:
1. Sign in as admin
2. Check a CC's profile and make sure there that next to the "Nominate Good Egg" option there is a "Nominate Escalation" option.
3. Click on Nominate Escalation and make sure that the options are the following: 

- TIER 1: Lobby This CC needs a little extra care! 
- TIER 2: Upstairs This CC is struggling, but receptive to feedback. 
- TIER 3: Mezzanine This CC is floundering, may be putting texters at risk, and/or not open to making changes. 
- TIER 4: Sky box Sound the alarm! We care about this CC, but this may not be the place for them.

#### Any migrations that will require the big-migrations tool?
(this includes `actor`, `conversation`, `conversation_participation`, `message`, `user` tables)

## Questions:
